client/packages/purchasing/src/purchase_order/ListView/ListView.tsx

```diff
@@ -110,6 +126,11 @@ const ListView: FC = () => {
         formatter: status =>
           getDeliveryStatusTranslator(t)(status as DeliveryStatus),
       },
+      {
+        key: 'targetMonths',
+        label: 'label.target-months',
+        accessor: ({ rowData }) => rowData.targetMonths,
+      },
```

Comment:
correct me if I'm wrong here but I'm thinking this should be a number cell? This will automatically handle display to 2dp, tooltip, and potential NaNs

```suggestion
 Cell: NumberCell,
      },
```